qldbdriver/session_management_test.go

```diff
@@ -36,6 +36,7 @@ func TestSessionManagementIntegration(t *testing.T) {
 	testBase := createTestBase()
 	testBase.deleteLedger(t)
 	testBase.createLedger(t)
+	defer testBase.deleteLedger(t)
```

Comment:
yesss, golang idioms!

TIL about [`T.Cleanup`](https://pkg.go.dev/testing@master#T.Cleanup) [[blog](https://ieftimov.com/posts/testing-in-go-clean-tests-using-t-cleanup/)] so this opens the door to a refactor of `createLedger` to have self-destructing ledgers.